exchange/binance.go

```diff
@@ -14,6 +15,8 @@ import (
 	"github.com/rodrigo-brito/ninjabot/tools/log"
 )
 
+var _ service.Feeder = &Binance{}
```

Comment:
What is the purpose of this validation, to force the implementation of the interface?